github-management/permissions.md

```diff
@@ -0,0 +1,100 @@
+# GitHub Permissions
+
+GitHub provides a limited permissions model for organizations and repositories.
+It lacks granularity, and for the most part is "all or nothing". This doesn't
+scale well with the size and velocity of the Kubernetes project.
+
+We have created a number of automated systems/bots to allow us to work around
+these limitations. Authorized users can issue
```

Comment:
link to our owners/approvers documentation?

Reply:
Done.